README.md

```diff
@@ -192,6 +192,14 @@ XFC.Provider.init({
 })
 ```
 
+If the app is using an alternate form a security and does require XFC to provide clickjacking support, a wildcard with no secret may be passed. Under these conditions, XFC will not hide the content and the consumer will automatically be authorized.
+
+```js
+XFC.Provider.init({
+  acls: ['*']
```

Comment:
Think it would be cleaner just to allow XFC.Provider.init() and not accepting any parameters assumes no security initialization.  Thoughts?